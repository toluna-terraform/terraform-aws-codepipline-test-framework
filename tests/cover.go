package test

import "log"

func check_cover(s string) {
		switch {
case s == "aws_codebuild_report_group.CodeCoverageReport[\"my_env\"]":
		log.Println("check coverage")
case s == "aws_codebuild_report_group.IntegrationTestReport[\"my_env\"]":
		log.Println("check coverage")
case s == "aws_codebuild_report_group.TestReport[\"my_env\"]":
		log.Println("check coverage")
case s == "aws_codebuild_project.tests_reports":
		log.Println("check coverage")
case s == "aws_iam_role.codebuild_role":
		log.Println("check coverage")
case s == "aws_iam_role_policy.codebuild_policy":
		log.Println("check coverage")
case s == "aws_iam_role_policy.codepipeline_policy":
		log.Println("check coverage")
case s == "aws_iam_role.test_framework":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-cloudwatch":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-codebuild":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-codedeploy":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-ec2":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-lambda-execution":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-lambda-ssm":
		log.Println("check coverage")
case s == "aws_iam_role_policy_attachment.role-s3":
		log.Println("check coverage")
case s == "aws_lambda_function.test_framework":
		log.Println("check coverage")
case s == "aws_lambda_layer_version.lambda_layer":
		log.Println("check coverage")
case s == "aws_s3_bucket.postman_bucket":
		log.Println("check coverage")
case s == "aws_s3_bucket_acl.postman_bucket":
		log.Println("check coverage")
case s == "aws_s3_bucket_policy.postman_bucket":
		log.Println("check coverage")
case s == "aws_s3_bucket_public_access_block.postman_bucket":
		log.Println("check coverage")
case s == "aws_s3_bucket_versioning.postman_bucket":
		log.Println("check coverage")
	}
}