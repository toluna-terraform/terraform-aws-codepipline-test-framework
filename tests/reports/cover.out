mode: set
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:7.1,8.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:9.1,10.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:11.1,12.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:13.1,14.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:15.1,16.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:17.1,18.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:19.1,20.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:21.1,22.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:23.1,24.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:25.1,26.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:27.1,28.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:29.1,30.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:31.1,32.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:33.1,34.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:35.1,36.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:37.1,38.0 1 1
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:39.1,40.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:41.1,42.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:43.1,44.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:45.1,46.0 1 1
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:47.1,48.0 1 0
github.com/toluna-terraform/terraform-aws-codepipline-test-framework/cover.go:49.1,50.0 1 0
